lib/ff_bot/github/merger.ex

```diff
@@ -66,4 +66,12 @@ defmodule FFBot.GitHub.Merger do
   defp push_changes(cloned) do
     {:ok, _} = Git.push(cloned)
   end
+
+  defp maybe_propaganda do
+    if Application.get_env(:ff_bot, :disable_propaganda?) != nil and :rand.uniform() < 0.01 do
```

Comment:
Should this conditional not be an equality check instead of inequality?

If `:disable_propaganda?` is set to anything *but* `nil`, we should go to the else clause.